doc/op_guide.md

```diff
@@ -23,16 +31,164 @@
+    >NOTE: It could take a few minutes for the pods to converge while persistent volumes are being
+    automatically provisioned and attached to the cluster nodes.
+
+    ```
+    kubectl get pods | grep cassandra-test
+    ```
+
+    ```
+    NAME                                          READY   STATUS             RESTARTS   AGE
+    cassandra-test-dc-cassandra-west1-a-0   2/2     Running            2          84m
+    cassandra-test-dc-cassandra-west1-b-0   2/2     Running            0          83m
+    cassandra-test-dc-cassandra-wesr1-c-0   2/2     Running            0          81m
+    ```
+ 1) Verify the Cassandra cluster is healthy:
+
+    ```
+    kubectl exec cassandra-test-dc-cassandra-west1-a-0 -c cassandra -- nodetool status
+    ```
+
+    ```
+    Datacenter: test-dc-cassandra
+    ==========================
+    Status=Up/Down
+    |/ State=Normal/Leaving/Joining/Moving
+    --  Address      Load       Tokens       Owns (effective)  Host ID                               Rack
+    UN  10.244.1.14  87.41 KiB  256          62.9%             dcf940c2-18d2-4a3a-8abf-833acadeca7e  west1-a
+    UN  10.244.2.9   87.38 KiB  256          69.6%             fd59fa32-aab0-485e-b04b-7ad4b75e54dd  west1-b
+    UN  10.244.0.10  69.91 KiB  256          67.5%             9e4883a1-e822-472f-920f-f2fc36c340c8  west1-c
+    ```
+
+ 1) Issue a sample query to the cluster:
+
+    ```
+    kubectl exec cassandra-test-dc-cassandra-west1-a-0 -c cassandra -- cqlsh -e "SELECT now() FROM system.local;" cassandra-test-dc-cassandra-nodes 
+    ```
+
+    ```
+     system.now()
+    --------------------------------------
+     243e2fd0-d64a-11e9-b8a4-2dd801fa1b1c
+
+    (1 rows)
+    ```
+### Resize a Cassandra cluster 
+
+ 1) Create a Cassandra cluster, if you haven't already:
+
+    ```bash
+    $ kubectl apply -f example/example-datacenter.yaml
+    ```
+
+ 1) In `example/example-datacenter.yaml` the initial cluster size is 3. Modify the file and change `replicas` from 3 to 5.
+
+    ```yaml
+    spec:
+      replicas: 5
+      image: "gcr.io/cassandra-operator/cassandra:latest"
+    ```
+
+ 1) Apply the size change to the cluster CR:
+    ```bash
+    $ kubectl apply -f example/example-datacenter.yaml
+    ```
+    The Cassandra cluster will scale to 5 members (5 pods):
+    ```bash
+    $ kubectl get pods
+    NAME                                    READY     STATUS    RESTARTS   AGE
+    cassandra-test-dc-cassandra-west1-a-0   2/2       Running   1          10m12s
+    cassandra-test-dc-cassandra-west1-a-1   2/2       Running   2          3m2s
+    cassandra-test-dc-cassandra-west1-b-0   2/2       Running   1          8m38s
+    cassandra-test-dc-cassandra-west1-b-1   2/2       Running   1          1m4s
+    cassandra-test-dc-cassandra-west1-c-0   2/2       Running   0          5m22s
+    ```
+
+ 1) Similarly we can decrease the size of cluster from 5 back to 3 by changing the size field again and reapplying the change.
```

Comment:
I would put here a note that it may take some time for cluster to react on this so scaling up / down is visible. Node is decommissioning in the background and takes some time to take it down and repeatedly do that does not make sense (even can do some harm maybe).